tests/fixtures/ansible_distribution-1.yml

```diff
@@ -57,7 +57,7 @@ interactions:
       User-Agent:
       - Python-urllib/3.9
     method: GET
-    uri: https://pulp.example.org/pulp/api/v3/contentguards/certguard/x509/?limit=1&name=test_content_guard
+    uri: https://pulp.example.org/pulp/api/v3/contentguards/?limit=1&name=test_content_guard
```

Comment:
Sorry, this is not the way to "adjust tests". Have you read about re-recording the pyvcr tapes in the README?

Reply:
I had an issue yesterday that meant the files would just be deleted and not rewritten when recording. That seems to have resolved itself overnight

Reply:
Should be fixed now